debvader/train.py

```diff
@@ -189,64 +212,67 @@ def kl_metric(y_true, y_pred):
             raise ValueError
 
     # Start from the weights of an already trained network (recommended if possible)
-    if from_survey is not None:
+    if (initial_weights_path is None) and (from_survey is not None):
 
-        data_path = pkg_resources.resource_filename("debvader", "data/")
-        path_output = os.path.join(
-            data_path, "weights/", str(from_survey), "not_normalised/"
+        initial_weights_path = pkg_resources.resource_filename("debvader", "data/")
+        initial_weights_path = os.path.join(
+            initial_weights_path, "weights/", str(from_survey)
```

Comment:
Make sure the paths are clean, the goal of path.join is to create the adequate link between folders depending on your current system.
I have shown several examples above.

Reply:
done!